internal/pkg/types/v1alpha2/steps.go

```diff
@@ -28,14 +28,15 @@ func (steps Steps) Validate() error {
 // Steps are executed sequentially, each step runs in its own
 // empty temporary directory.
 type Step struct {
-	Env       Environment  `yaml:"env,omitempty"`
-	CachePath string       `yaml:"cache,omitempty"`
-	TmpDir    string       `yaml:"-"`
-	Sources   Sources      `yaml:"sources,omitempty"`
-	Prepare   Instructions `yaml:"prepare,omitempty"`
-	Build     Instructions `yaml:"build,omitempty"`
-	Install   Instructions `yaml:"install,omitempty"`
-	Test      Instructions `yaml:"test,omitempty"`
+	Env        Environment  `yaml:"env,omitempty"`
+	CachePath  string       `yaml:"cache,omitempty"`
```

Comment:
should we remove this or just deprecate it, it's not used by any projects under siderolabs

Reply:
was not even implemented before?

Reply:
it was, but for some reason only supported a single cache path, while in theory buildkit supports multiple